.github/workflows/package-fedora.yml

```diff
@@ -0,0 +1,38 @@
+name: Package GEDKeeper3 for Fedora
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  DOTNET_VERSION: '6.0.*'
+
+jobs:
+  build-fedora:
+    runs-on: ubuntu-latest
```

Comment:
I think this actually needs to be built `in` Fedora, not on Ubuntu.

Reply:
I think not need any specific packages for separate build this project in Fedora.